clients/drcachesim/common/trace_entry.h

```diff
@@ -431,6 +431,12 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_RSEQ_ENTRY,
 
```

Comment:
TRACE_MARKER_TYPE_RSEQ_ABORT is always followed by TRACE_MARKER_TYPE_KERNEL_EVENT. What address does the kernel event marker have?

Reply:
The same abort handler PC.  It does seem redundant; but simpler to leave it I think than possibly break someone by changing (though we're already changing the abort marker value...)